packages/@o3r/testing/src/core/playwright/element.ts

```diff
@@ -68,26 +68,14 @@ export class O3rElement implements ElementProfile {
 
   /** @inheritdoc */
   public async setValue(input: string) {
-    // Clears the value in the field
-    await this.clearValue();
-
     await this.sourceElement.element.fill(input);
 
     await this.sourceElement.element.press('Tab');
   }
 
   /** @inheritdoc */
   public async clearValue() {
```

Comment:
Wasn't it working before ? Does the simulation of a user deleting character by character had an utility compare to the fill with blank space ?

Reply:
It is now pretty buggy, there seems to be some timing issues and if I deactivate the headless mode it is really obvious the behaviour is not quite deterministic.